scss/_patterns_tooltips.scss

```diff
@@ -53,12 +53,26 @@ $triangle-height: $sp-unit;
       position: absolute;
       width: 0;
     }
+
+    &::after {
+      border-radius: 10% 90% 0% 100% / 100% 100% 0% 0%;
+      content: '';
+      height: 15px;
+      left: 0;
+      position: absolute;
+      right: 0;
+      top: -15px;
+    }
   }
 
   // Bottom center tooltip
   .p-tooltip--btm-center {
     @extend %tooltip;
 
+    &:hover::before {
```

Comment:
Is this needed? It doesn't seem to do anything.
It's targeting `.p-tooltip--btm-center:hover::before`, but we don't have `::before` defined for it, so these styles don't target any element.
It's redefined for all position variants of the tooltip.

Reply:
Or is it a leftover of previous implementation?

Reply:
Sorry, that's left over from the previous solution - updated and removed!